service/src/alerts/index.ts

```diff
@@ -12,10 +12,12 @@ export async function alertOnResult({
     testFiles,
     results,
     testVariables,
+    runId,
 }: {
     testFiles: Record<string, string>
     results: EnhancedAggregatedResult
     testVariables: Partial<Record<string, string>>
+    runId: string,
```

Comment:
```suggestion
    runId: string
```

L20 is a typescript type which uses semicolons or newlines as separators. So you can just remove the comma

Reply:
Thanks @noahnu 🙏 , test is passing now.